modules/ROOT/pages/software-extensibility.adoc

```diff
@@ -16,26 +16,23 @@ These extension points are guaranteed to be stable across versions of Bonita 7:
 
 === Connectors
 
-In Community, Teamwork, Efficiency, Performance and Enterprise editions
-
 Bonita comes with more than 80 xref:connectivity-overview.adoc[standard connectors] to the major information system components: major databases (Oracle, Microsoft, Postgres, etc.), SOAP Webservice, Salesforce, Email, etc.
-A new connector enables you to provide new connectivity capabilities for processes.
-To xref:connectors-overview.adoc[implement a new connector], you need to provide some XML description files and a
-Java class respecting the https://javadoc.bonitasoft.com/api/{javadocVersion}/index.html[Connector interface].
-Connectors can be implemented directly from Bonita Studio.
 
-=== Actor filters
+However you create your own connectors to have new connectivity capabilities for processes.
 
-In Community, Teamwork, Efficiency, Performance and Enterprise editions
+Connectors are developped outside the Studio with a dedicated maven archetype. Consult this page for more information: xref:connector-archetype.adoc[Create a connector].
+
+Custom connectors are added to a Bonita project from the xref:managing_extension_studio.adoc[extension view].
+
+=== Actor filters
 
 Bonita comes with a set of xref:actor-filtering.adoc[standard actor filters] that can be used to reduce the list of candidates for process tasks.
-A new actor filter enables you to provide new filtering capabilities to processes. To xref:creating-an-actor-filter.adoc[implement a new actor filter],
```

Comment:
To use one of them in the processes, go to the Extensions view and select the actor filter in the "Marketplace".
Then, configure it at human task level in the "General" tab.